tests/store/conftest.py

```diff
@@ -149,9 +149,7 @@ def fixture_microbial_store(
         application_version = base_store.get_application_by_tag(
             sample_data["application"]
         ).versions[0]
-        organism = base_store.Organism(
-            internal_id=sample_data["organism"], name=sample_data["organism"]
-        )
+        organism = Organism(internal_id=sample_data["organism"], name=sample_data["organism"])
```

Comment:
Type hint

Reply:
Added it, but I think it makes the code less readable in this instance